requirements.txt

```diff
@@ -0,0 +1,11 @@
+ftfy==4.4.3; python_version < '3.0'
+ftfy>= 5.0.0; python_version >= '3.0'
+joblib
+numpy
+pandas
+sklearn
+spacy
+tensorflow
```

Comment:
The code in this repo is not compatible with the latest TF, so some restriction here is needed.